src/cli.ts

```diff
@@ -44,6 +45,16 @@ yargs
     ),
     async (argv) => { handleStats(argv) }
   )
+  .command('split [entrypoint]', 'Split document references into files',
+    (yargs) => yargs
+      .positional('entrypoint', { type: 'string' })
+      .option({ outDir: {
```

Comment:
outDir seems like we have to pay a price per character. How about `outputDirectory` ?

Reply:
@adamaltman 
let's leave **outDir**, because it's a common practice.
Please have a look: 
https://www.typescriptlang.org/docs/handbook/compiler-options.html
https://www.typescriptlang.org/docs/handbook/migrating-from-javascript.html#writing-a-configuration-file

Reply:
Thank you!